### PR TITLE
Subscription approval docs

### PR DESCRIPTION
**Pull request by jhivandb**
Resolves:
- https://github.com/wso2-enterprise/choreo/issues/30591

### CODE REVIEW

File: en/docs/administer/create-api-subscription-plans.md
Comment (Nashaath) [inline en/docs/administer/create-api-subscription-plans.md]: Suggestion:
- Select the **Approval required** checkbox if subscription requests made to this plan require administrator or API publisher approval to activate the plan. This allows for manual review and approval of a subscription before granting API access.

File: en/docs/api-management/manage-api-traffic/subscribe-to-an-api-with-a-subscription-plan.md
Comment (Nashaath) [inline en/docs/api-management/manage-api-traffic/subscribe-to-an-api-with-a-subscription-plan.md]: Shall we update step 7 as follows since the button name differs if approval enabled:

7. If the selected subscription plan requires administrator or API publisher approval to become active, click **Request Subscription**. Otherwise, click **Add Subscription**. If the selected subscription plan requires approval, your subscription will be shown as "Pending" until it is approved by an organization admin or the API publisher.

    !!! Note
         If the subscription plan requires administrator or API publisher approval to become active, the subscription status will be set to **Pending Creation** until it is reviewed and approved.

File: en/docs/api-management/manage-api-traffic/configure-subscription-approval.md
Comment (Nashaath) [inline en/docs/api-management/manage-api-traffic/configure-subscription-approval.md]: We might not need this page because I have added the necessary info accordingly in en/docs/administer/configure-approvals-for-choreo-workflows.md.
WDYT?

**Review by Nashaath — APPROVED** [review]

File: en/docs/administer/configure-approvals-for-choreo-workflows.md
Comment (malinthaprasan) [inline en/docs/administer/configure-approvals-for-choreo-workflows.md:26]: `**Approve API subscriptions**: Grants access to approve or reject API subscription requests.`
This needs to be under WORKFLOW-MANAGEMENT set, doesn't it? 

And re: "Environment promotion" I think we need to have a similar way to indicate "Approve component promotion requests" too

**Pull request merged**
